Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantSedin.cs

```diff
@@ -0,0 +1,49 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Content.Server.Popups;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class PlantSedin : ReagentEffect
```

Comment:
documentation

Reply:
I have no idea how to do this, can you explain?

Reply:
> I have no idea how to do this, can you explain?

`/// <summary>`
`///    Explanation goes here`
`/// </summary>`